src/control/cmd/daos_agent/config_test.go

```diff
@@ -62,6 +62,7 @@ fabric_ifaces:
   -
      iface: ib3
      domain: mlx5_3
+provider: ofi+tcp
```

Comment:
nit: indenting issue?

Reply:
It's intended to be on its own, not part of the `fabric_ifaces` group above. I think it just looks weird with the limited context of the diff.